arch-secure-boot

```diff
@@ -12,8 +12,13 @@ ESP="${ESP:-/efi}"
 EFI="${EFI:-/EFI/arch}"
 KERNEL="${KERNEL:-linux}"
 NAME="${NAME:-secure-boot-$KERNEL}"
+SNAPPER_OPTIONS="${SNAPPER_OPTIONS:---no-dbus}"
```

Comment:
> First, I had to remove `--no-dbus` option from snapper comand-line because with it, it was unable to find my snapshots.

This is so strange, I used this flag even in `grub-btrfs` (it's still present there) and I've never seen anyone report such issue! I wonder why dbus is required for you to list snapshots. I also use btrfs (single disk with subvolumes) and snapper, and I'm just wondering if this is somehow very special to your own setup, maybe some special configuration that you have... Maybe something that is worth investigating, before adding a new option?